Packs/DigitalShadows/Integrations/ReliaQuestGreyMatterDRPEventCollector/ReliaQuestGreyMatterDRPEventCollector.yml

```diff
@@ -0,0 +1,76 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: ReliaQuest GreyMatter DRP Event Collector
+  version: -1
+configuration:
+- display: Server URL
+  name: url
+  required: true
+  type: 0
+  defaultvalue: https://api.searchlight.app
+- display: Username
+  name: credentials
+  type: 9
+  required: true
+  section: Connect
+- display: Account ID
+  name: account_id
+  type: 0
+  required: true
+  section: Connect
+- display: Maximum number of events per fetch
+  name: max_fetch_events
+  type: 0
+  defaultvalue: '200'
+  required: true
+  section: Collect
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+  section: Connect
+  advanced: true
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+  section: Connect
+  advanced: true
+description: ReliaQuest GreyMatter DRP Event Collector monitors and manages an organization's digital risk across the widest range of data sources within the open, deep, and dark web.
+display: ReliaQuest GreyMatter DRP Event Collector
+name: ReliaQuest GreyMatter DRP Event Collector
+script:
+  commands:
+  - arguments:
+    - name: start_time
+      description: 'From which time to get the events in ISO8601 format, for example 2020-09-24T16:30:10.016Z. If not provided, will retrieve the oldest events available.'
```

Comment:
lets use dateparse so dont need such an exact intake

Reply:
the truth is i already use the date parser so ill just update the doc-string here.